src/composables/store.ts

```diff
@@ -50,15 +53,17 @@ export const useStore = (initial: Initial) => {
   const prUrl = `https://preview-${pr}-element-plus.surge.sh/bundle/dist`
 
   const versions = reactive<Versions>({
-    vue: 'latest',
-    elementPlus: pr ? 'preview' : 'latest',
-    typescript: 'latest',
+    vue: saved?._o?.vueVersion ?? 'latest',
+    elementPlus: pr
+      ? 'preview'
+      : saved?._o?.elVersion ?? 'latest',
+    typescript: saved?._o?.tsVersion ?? 'latest',
   })
   const userOptions: UserOptions = pr
     ? {
-        showHidden: true,
-        styleSource: `${prUrl}/index.css`,
-      }
+      showHidden: true,
+      styleSource: `${prUrl}/index.css`,
+    }
     : {}
 
```

Comment:
seems like it cannot be persistent between reload's

```suggestion
 Object.assign(userOptions, {
   vueVersion: saved?._o?.vueVersion,
   tsVersion: saved?._o?.tsVersion,
   elVersion: saved?._o?.elVersion,
 })
```

Reply:
Thank you for your advice😊I've already changed it